.azure-pipelines.yml

```diff
@@ -121,3 +122,29 @@ jobs:
   - template: tools/ci/azure/tox_pytest.yml
     parameters:
       directory: tools/wpt/
+
+- job: all_macOS
+  displayName: 'all tests (Safari Technology Preview)'
+  condition: eq(variables['Build.SourceBranch'], 'refs/heads/azure-daily-safari')
```

Comment:
Looks like (capacity allowing,) this would be easy to make run on all master commits by tweaking this condition.

Reply:
Indeed!